docs/repo.toml

```diff
@@ -15,6 +15,8 @@ social_media     = [
     [ "discord", "https://discord.com/channels/1019361803752456192/1161051667945508884" ],
 ]
 
+sphinx_version = "4.5.0.2-py3.10-${platform}"
```

Comment:
FWIW, this tag (sphinx 4.5) appears to be from March 2022. I know this may be constrained by other parts of the docs builds, but just wanted to note that it's fairly old.